Source/Core/Core/State.cpp

```diff
@@ -413,6 +413,9 @@ static void CompressAndDumpState(CompressAndDumpState_args& save_args)
   else
     f.WriteBytes(buffer_data, buffer_size);
 
+  if (!f.IsGood())
+    Core::DisplayMessage("Failed to write temporary state file", 2000);
```

Comment:
From the user's perspective, the savestate is most likely not temporary, so this might be a bit confusing. But I'm not sure what we should do about it – it is technically correct after all.

Reply:
I removed the word "temporary" from this and the other messages.